<testsuites>
	<testsuite id="1" tests="1" failures="1" package="/Users/alper/Desktop/taskloom/taskloom-api-test/specs/task_specs/task_delete.spec" time="0.007" timestamp="2025-09-08T15:27:39" name="Delete Task" errors="0" hostname="ip-10-1-32-36.eu-central-1.compute.internal">
		<properties></properties>
		<testcase classname="Delete Task" name="Delete task by id" time="0.007">
			<failure message="Response status code should be &#34;204&#34;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;204&gt; but was: &lt;404&gt;&#39;" type="Response status code should be &#34;204&#34;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;204&gt; but was: &lt;404&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;204&gt; but was: &lt;404&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:35)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
	<testsuite id="2" tests="2" failures="1" package="/Users/alper/Desktop/taskloom/taskloom-api-test/specs/task_specs/task_get_list.spec" time="0.018" timestamp="2025-09-08T15:27:39" name="Get Tasks" errors="0" hostname="ip-10-1-32-36.eu-central-1.compute.internal">
		<properties></properties>
		<testcase classname="Get Tasks" name="Get task by id" time="0.007">
			<failure message="Response status code should be &#34;200&#34;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;404&gt;&#39;" type="Response status code should be &#34;200&#34;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;404&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;404&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:35)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<testcase classname="Get Tasks" name="Get all tasks" time="0.011"></testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
	<testsuite id="3" tests="1" failures="1" package="/Users/alper/Desktop/taskloom/taskloom-api-test/specs/task_specs/task_status.spec" time="0.008" timestamp="2025-09-08T15:27:39" name="Update Task Status" errors="0" hostname="ip-10-1-32-36.eu-central-1.compute.internal">
		<properties></properties>
		<testcase classname="Update Task Status" name="Patch task status" time="0.008">
			<failure message="Response status code should be &#34;202&#34;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;404&gt;&#39;" type="Response status code should be &#34;202&#34;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;404&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;404&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:35)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
	<testsuite id="4" tests="4" failures="1" package="/Users/alper/Desktop/taskloom/taskloom-api-test/specs/task_specs/taks_update.spec" time="0.411" timestamp="2025-09-08T15:27:39" name="Update Task" errors="0" hostname="ip-10-1-32-36.eu-central-1.compute.internal">
		<properties></properties>
		<testcase classname="Update Task" name="Update task with valid arguments" time="0.385">
			<failure message="Response status code should be &#34;202&#34;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;400&gt;&#39;" type="Response status code should be &#34;202&#34;&#xA;Step Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;400&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;400&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:35)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<testcase classname="Update Task" name="Update task with incorrect description" time="0.011"></testcase>
		<testcase classname="Update Task" name="Update Task with not exist user id" time="0.007"></testcase>
		<testcase classname="Update Task" name="Update Task without user id" time="0.008"></testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
	<testsuite id="5" tests="4" failures="0" package="/Users/alper/Desktop/taskloom/taskloom-api-test/specs/task_specs/task_create.spec" time="0.208" timestamp="2025-09-08T15:27:39" name="Create Task" errors="0" hostname="ip-10-1-32-36.eu-central-1.compute.internal">
		<properties></properties>
		<testcase classname="Create Task" name="Create task with status" time="0.171"></testcase>
		<testcase classname="Create Task" name="Create Task without status response should be TODO" time="0.020"></testcase>
		<testcase classname="Create Task" name="Create Task with invalid user id" time="0.009"></testcase>
		<testcase classname="Create Task" name="Create Task without user id" time="0.008"></testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
</testsuites>